### PR TITLE
Banner with version and CRC32 sum of program files

### PR DESCRIPTION
**Pull request by babenek**
## Description

- Added `--banner` option to distinguish various version in development. CRC32 sum of files might be displayed in banner at program start. ```CredSweeper v1.4.2 crc32:ae1baf99```. It is not an integrity check but fast.
- Use natural order in benchmark first:old, second:new.

## How has this been tested?

- [x] Add `--banner` to arguments. See crc32 value. 
- [x] Check action 'Static analysis and code style' step 'Integrity and version check'

### CODE REVIEW

File: credsweeper/__init__.py [hunk 1]
```diff
@@ -8,8 +6,6 @@
 from credsweeper.ml_model.ml_validator import MlValidator
 from credsweeper.validations.apply_validation import ApplyValidation
 
-os.environ["TF_CPP_MIN_LOG_LEVEL"] = "2"
-
```
Comment (csh519) [inline credsweeper/__init__.py:12]: As I know this line is needed to avoid warning log..
Could you explain why this line should be removed?

Comment (babenek) [reply credsweeper/__init__.py:12]: The environment used to manage log in tensorflow package. Since we use onnxruntime - it is extra.

Comment (csh519) [reply credsweeper/__init__.py:12]: Oh.. That's right. Thank you for remove useless lines.

**Review by csh519 — APPROVED** [review]
Thank you for adding `--banner` option for checksum.

LGTM :+1:

**Review by iuriimet — APPROVED** [review]
LGTM

**Review by kmnls — APPROVED** [review]

**Pull request merged**